Dockerfile

```diff
@@ -12,47 +12,53 @@ ENV PATH /tmp/google-cloud-sdk/bin:$PATH
+RUN curl --location https://download.docker.com/linux/static/edge/x86_64/docker-$DOCKER_CLIENT_VERSION.tgz --output /tmp/docker-$DOCKER_CLIENT_VERSION.tgz && \
         tar --extract --gzip --directory /tmp --file /tmp/docker-$DOCKER_CLIENT_VERSION.tgz && \
         mv /tmp/docker/* /usr/bin && \
-        rm -rf /tmp/* && \
-        curl --location https://releases.hashicorp.com/terraform/${TERRAFORM_VERSION}/terraform_${TERRAFORM_VERSION}_linux_amd64.zip --output terraform.zip && \
+        rm -rf /tmp/*
+
+# Install Terraform
+RUN curl --location https://releases.hashicorp.com/terraform/${TERRAFORM_VERSION}/terraform_${TERRAFORM_VERSION}_linux_amd64.zip --output terraform.zip && \
         unzip terraform.zip && \
         mv terraform /usr/bin/terraform && \
-        rm -rf terraform.zip && \
-        curl --location https://storage.googleapis.com/kubernetes-release/release/v$KUBECTL_VERSION/bin/linux/amd64/kubectl --output /usr/bin/kubectl && \
-        chmod +x /usr/bin/kubectl && \
-        wget --output-document=/usr/local/bin/yq https://github.com/mikefarah/yq/releases/download/$YQ_VERSION/yq_linux_amd64 && \
-        chmod +x /usr/local/bin/yq && \
-        curl --location https://github.com/kubernetes-sigs/kustomize/releases/download/v${KUSTOMIZE_VERSION}/kustomize_${KUSTOMIZE_VERSION}_linux_amd64 --output /usr/local/bin/kustomize && \
+        rm -rf terraform.zip
+
+# Install Kubectl
+RUN curl --location https://storage.googleapis.com/kubernetes-release/release/v$KUBECTL_VERSION/bin/linux/amd64/kubectl --output /usr/bin/kubectl && \
+        chmod +x /usr/bin/kubectl
+
+# Install yq
+RUN wget --output-document=/usr/local/bin/yq https://github.com/mikefarah/yq/releases/download/$YQ_VERSION/yq_linux_amd64 && \
+        chmod +x /usr/local/bin/yq
+
+# Install Kustomize
+RUN curl --location https://github.com/kubernetes-sigs/kustomize/releases/download/v${KUSTOMIZE_VERSION}/kustomize_${KUSTOMIZE_VERSION}_linux_amd64 --output /usr/local/bin/kustomize && \
         chmod +x /usr/local/bin/kustomize
 
-RUN apk --no-cache add \
-        curl \
-        python \
-        py-crcmod \
-        bash \
-        libc6-compat \
-        openssh-client \
-        git \
-    && curl -O https://dl.google.com/dl/cloudsdk/channels/rapid/downloads/google-cloud-sdk-${CLOUD_SDK_VERSION}-linux-x86_64.tar.gz && \
-    tar xzf google-cloud-sdk-${CLOUD_SDK_VERSION}-linux-x86_64.tar.gz && \
-    rm google-cloud-sdk-${CLOUD_SDK_VERSION}-linux-x86_64.tar.gz && \
-    ln -s /lib /lib64 && \
-    gcloud config set core/disable_usage_reporting true && \
-    gcloud config set component_manager/disable_update_check true && \
-    gcloud config set metrics/environment github_docker_image && \
-    gcloud --version
+# Install Google Cloud SDK
+RUN curl -O https://dl.google.com/dl/cloudsdk/channels/rapid/downloads/google-cloud-sdk-${CLOUD_SDK_VERSION}-linux-x86_64.tar.gz && \
+        tar xzf google-cloud-sdk-${CLOUD_SDK_VERSION}-linux-x86_64.tar.gz && \
```

Comment:
This, along with `ENV PATH /tmp/google-cloud-sdk/bin:$PATH`, installs it directly into the workspace `/tmp`. Other tools were moved into `/usr/bin`, using eg. `mv /tmp/docker/* /usr/bin`, `mv terraform /usr/bin/terraform`